nixos/modules/misc/ids.nix

```diff
@@ -270,6 +270,7 @@
       toxvpn = 247;
       squeezelite = 248;
       turnserver = 249;
+      dunst = 250;
```

Comment:
Doesn't look like these are ever used in the configuration, am I missing something?